frontend/src/scenes/dashboard/dashboardLogic.js

```diff
@@ -399,5 +398,24 @@ export const dashboardLogic = kea({
                 setTimeout(() => actions.refreshDashboardItem(id), 1000)
             }
         },
+        reportUsage: async (payload, breakpoint) => {
```

Comment:
What's the type for payload? I assume it's defaulting to `any` right now.

Reply:
Well this file is still JS so no typing 😢

Reply:
aha :D nvm then